Core/Debugger/WebSocket/GameSubscriber.cpp

```diff
@@ -96,6 +96,18 @@ void WebSocketGameStatus(DebuggerRequest &req) {
 //  - version: string, typically starts with "v" and may have git build info.
 void WebSocketVersion(DebuggerRequest &req) {
 	JsonWriter &json = req.Respond();
+
+	std::string version;
+	if (!req.ParamString("version", &version)) {
+		return;
+	}
+	std::string name;
+	if (!req.ParamString("name", &name)) {
+		return;
+	}
```

Comment:
Hm, this doesn't seem backward compatible with any old client that doesn't pass its own version/name info, but just wants to know PPSSPP's version.

Reply:
Quoting my old PR:
> Before this change, the version event had de-facto 2 optional arguments, which are now mandatory as the description claims.

Reply:
Okay, your old PR states that, but what's the motivation for making them mandatory?

Reply:
They were supposed to be mandatory (the description doesn't mention them being optional) and I think there were no means of actually storing the info.
You can check this: https://github.com/hrydgard/ppsspp/blob/7f0828c6595f60c7dba02643452933a061a318ec/Core/Debugger/WebSocket.cpp#L43
It's a warning for all people who want to use the API. The details on how the name and the version may be later used, by the way, can be found in our Discord.

Reply:
Right, but as a reviewer, I'm not always going to have all that information in my head, I don't remember everything that's been discussed or has been commented in another file. 

So tip for next time: When proposing a change that looks obviously backwards incompatible, you really REALLY need to remind the reviewer of why it's OK to make the change, either through a comment directly in the code, or as part of the pull request.

Reply:
I'm not entirely sure it's necessary to make these required, as you could imagine this being called just to get the version again.  It is the suggested initial handshake.

It's also not clear that all the this isn't a breaking change for any private clients a rom hacker or similar might use, and it's not clear there's actual motivation for that right now.  As previously described, this is more about making it possible to handle future breaking changes rather than just a vehicle through which to introduce breaking change.

-[Unknown]

Reply:
Hm, right. If those are not supplied, we should:

 * If version is not already set, set to some kind of oldest marker if the fields are missing? Or don't do anything at all.
 * Otherwise, just return the ppsspp version

I'll change it to that.